security/pfSense-pkg-acme/Makefile

```diff
@@ -0,0 +1,97 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-acme
+PORTVERSION=	0.1
+CATEGORIES=	security
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	PiBa-NL
+COMMENT=	pfSense package acme
+
+USE_PHP= ftp
+
+CONFLICTS=	
+
+NO_BUILD=	yes
+NO_MTREE=	yes
+
+SUB_FILES=	pkg-install pkg-deinstall
+SUB_LIST=	PORTNAME=${PORTNAME}
+
+do-extract:
+	${MKDIR} ${WRKSRC}
+
+do-install:
+	${MKDIR} ${STAGEDIR}${PREFIX}/pkg
+	${MKDIR} ${STAGEDIR}${PREFIX}/pkg/acme
+	${MKDIR} ${STAGEDIR}${PREFIX}/pkg/acme/dnsapi
```

Comment:
MKDIR macro is 'mkdir -p' so you just need to run it to last level and it will create parent directories, remove lines creating pkg and pkg/acme